server/models/ExperimentAttributesValues.ts

```diff
@@ -0,0 +1,31 @@
+import { DataTypes, Model } from "sequelize"
+import sequelize from "../utils/sequelize"
+import ExperimentAttributes from "./ExperimentAttributes"
+
+class ExperimentAttributesValues extends Model {}
+
+ExperimentAttributesValues.init(
+  {
+    // Model attributes are defined here
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+      references: {
```

Comment:
Sorry I missed it the first time, but the reference would need to be an extra field and we also said we do them in a separate issue. So just remove the references here.